### PR TITLE
Updates postures.

### PR DESCRIPTION
**Pull request by diekus**
👀 @darktears. Trying to abstract the postures to futureproof them (and not only make them dependable on foldable devices).

***
<a href="https://pr-preview.s3.amazonaws.com/diekus/device-posture/pull/83.html" title="Last updated on Jul 5, 2021, 2:43 PM UTC (ecf936d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/device-posture/83/c5380a1...diekus:ecf936d.html" title="Last updated on Jul 5, 2021, 2:43 PM UTC (ecf936d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,9 +128,7 @@ <h2>
         Introduction
       </h2>
       <p>
-        New types of mobile devices are appearing that have some sort of folding capabilities around the screen. Hence, 
-        when the device folds, it physically can be different form factor. The main interest in knowing the posture is to 
-        enable new user experiences with responsive design.
+        The device posture is the physical position in which a device holds which may be derived from sensors in addition to the angle. New types of mobile devices are appearing that have some sort of capabilities that allow them to change their posture. The most common types of devices are the ones that can fold (their screen or around their screen), allowing them to physically alter their form factor. The main interest in knowing the posture of a device is to enable new user experiences with responsive design.
```
Comment (kenchris) [inline index.html]: Please break lines... you can use tidy if you want - this is unreviewable on Github :-(

File: index.html [hunk 2]
```diff
@@ -224,35 +238,37 @@ <h2>
...
-          <a>Laptop</a> posture indicates that the device is being used as
-          a traditional laptop, meaning one screen is placed on a more or
-          less horizontal surface with a screen angle between 180 to 0 degrees.
-        </li>
-        <li>
-          <img src="images/flat-mode-graphic.svg" class="drawing" alt="drawing of flat mode">
-          <a>Flat</a> posture indicates that one screen is being placed on
-          a more or less horizontal surface with a screen angle around 180 degrees.
+          <img src="images/continuos-posture.svg" class="drawing" alt="drawing of the continuous posture">
+          <p><a>Continuous</a> posture: The continuous posture refers to a "flat" position. This is the default case.</p>
```
Comment (kenchris) [inline index.html]: to a flat, folded out position. This is the default case for most devices not allowing different postures

File: index.html [hunk 3]
```diff
@@ -224,35 +238,37 @@ <h2>
...
+          <p>It includes devices with no folds, hinges or similar capabilities.</p>
+          <p>Due to the nature of hardware, it also includes devices with dual, foldable, rollable or curved screens, as long as they are in a posture where the document is expected to be displayed with a flat layout.</p>
```
Comment (kenchris) [inline index.html]: hardware innovation

File: index.html [hunk 4]
```diff
@@ -224,35 +238,37 @@ <h2>
...
+          <p>Examples of these are:</p>
+          <ul>
+            <li>Foldable device in a flat posture.</li>
```
Comment (kenchris) [inline index.html]: >Foldable device in a flat, fully unfolded posture.

File: index.html [hunk 5]
```diff
@@ -224,35 +238,37 @@ <h2>
...
+            <li>Foldable device running the browser in a window/section that does not span the hinge.</li>
```
Comment (kenchris) [inline index.html]: span across

**Review by kenchris — APPROVED** [review]
Please fix the comments

File: index.html (hunk 1 above)
Comment (diekus) [reply index.html]: I'll run it through tidy.

**Comment by diekus** [comment]
@kenchris all comments have been fixed!

**Review by kenchris — APPROVED** [review]

**Comment by diekus** [comment]
☠️

**Pull request closed without merging**